src/internal/connector/onedrive/item_test.go

```diff
@@ -106,73 +122,90 @@ func (suite *ItemIntegrationSuite) TestItemReader() {
 		driveItemID,
 		"no file item found for user %s drive %s",
 		suite.user,
-		suite.driveID,
+		suite.userDriveID,
 	)
 
 	// Read data for the file
 
-	itemInfo, itemData, err := driveItemReader(ctx, suite, suite.driveID, driveItemID)
+	itemInfo, itemData, err := oneDriveItemReader(ctx, suite, suite.userDriveID, driveItemID)
 	require.NoError(suite.T(), err)
-	require.NotNil(suite.T(), itemInfo)
-	require.NotEmpty(suite.T(), itemInfo.ItemName)
+	require.NotNil(suite.T(), itemInfo.OneDrive)
+	require.NotEmpty(suite.T(), itemInfo.OneDrive.ItemName)
 
 	size, err := io.Copy(io.Discard, itemData)
 	require.NoError(suite.T(), err)
 	require.NotZero(suite.T(), size)
-	require.Equal(suite.T(), size, itemInfo.Size)
-	suite.T().Logf("Read %d bytes from file %s.", size, itemInfo.ItemName)
+	require.Equal(suite.T(), size, itemInfo.OneDrive.Size)
```

Comment:
Unsure if this is the best metric. I believe @ashmrtn had a few issues where byte size within the store would vary. Not sure that a variance between these numbers would necessarily be an error.

Reply:
The test has been consistently successful so far.  I don't see reason to change it.